packages/pn-pa-webapp/src/navigation/__test__/RequireAuth.test.tsx

```diff
@@ -55,3 +57,26 @@ describe('RequireAuth Component', () => {
     expect(results).toHaveNoViolations();
   });
 });
+
+describe('RequireAuth Component without token', () => {
```

Comment:
Invece di creare due suit diverse, proverei ad unirli per evitare codice replicato.
Nella funzione di render di può passare anche un componente creato nel test. L'idea è quella di creare un componente (nel test) a cui passare il valore del token in fase di rendering.
Per esempio vedere App.test.tsx di pn-pa